docs/content/preview/explore/change-data-capture/_index.md

```diff
@@ -52,11 +48,13 @@ The current YugabyteDB CDC implementation supports only Debezium and Kafka.
 
 {{< warning title="Warning" >}}
 
-YugabyteDB doesn't yet support the DROP TABLE and TRUNCATE TABLE commands. The behavior of these commands while streaming data from CDC is undefined. If you need to drop or truncate a table, delete the stream ID using [yb-admin](../../admin/yb-admin/#change-data-capture-cdc-commands). See the [limitations](#limitations) section, as well.
+YugabyteDB doesn't yet support the DROP TABLE and TRUNCATE TABLE commands. The behavior of these commands while streaming data from CDC is undefined. If you need to drop or truncate a table, delete the stream ID using [yb-admin](../../admin/yb-admin/#change-data-capture-cdc-commands). See also the [limitations](#limitations) section.
```

Comment:
Suggestion:
See also the [limitations](#limitations) section.  >  See also [limitations](#limitations).